internal/symbolizer/symbolizer_test.go

```diff
@@ -149,11 +167,28 @@ func symbolzMock(sources plugin.MappingSources, syms func(string, string) ([]byt
 	return nil
 }
 
-func localMock(mode string, p *profile.Profile, obj plugin.ObjTool, ui plugin.UI) error {
-	p.Comments = append(p.Comments, "local="+mode)
+func localMock(p *profile.Profile, fast, force bool, obj plugin.ObjTool, ui plugin.UI) error {
+	var args []string
+	if fast {
+		args = append(args, "fast")
+	}
+	if force {
+		args = append(args, "force")
```

Comment:
Maybe use "force-local" string here to make the wanted comment string in the test cases a bit clearer?

Reply:
local=force-local seems redundant to me. Can we leave as is?

Reply:
Was looking to make the string in tests a bit more readable (currently have to know that the first force is for symbolization and the second for demangling), but leaving as is SGTM, it's a minor thing.